app/src/main/java/org/mozilla/fenix/customtabs/CustomTabActivity.kt

```diff
@@ -6,6 +6,6 @@ package org.mozilla.fenix.customtabs
 
 import org.mozilla.fenix.HomeActivity
 
-class CustomTabActivity : HomeActivity() {
+open class CustomTabActivity : HomeActivity() {
```

Comment:
Do we want all present and future rules from custom tabs to apply to this new account observer activity? It's certainly easier in code now to inherit everything, but we must decide if it'll cause maintenance headaches later.

Reply:
I think so? It will just be a simplified Custom Tab and I'm assuming any future custom tab additions will be optional args?

Reply:
@colintheshots @ekager I think this is a good step forward until we can actually give UX time to think about this flow 👍